### ISSUE TITLE
can't find node core 'event' module

### ISSUE THREAD

**Issue by davepacheco** [facebook/flow#88]
Sorry if I'm doing something totally wrong, but flow can't seem to find some built-in Node modules.  Here's my example, from scratch:

```
$ find .
.
./test.js

$ cat test.js
var events = require('events');
console.log(events.EventEmitter.name);

$ flow --version
Flow, a static type checker for Javascript, version 0.1.1

$ flow init

$ flow check --all

/Users/dap/work/tools/flowtest/test.js:1:14,30: events
Required module not found

Found 1 error

$ node test.js 
EventEmitter
```

**Comment by mroch** [comment]
looks this got added at some point

https://github.com/facebook/flow/blob/656496a7f6ea65f71e9f655d19d5072f4cca71f2/lib/node.js#L439

**Comment by gasi** [comment]
Are the Node.js interfaces automatically loaded? I tried `flow` 19.1 out of the box:

```
> flow version
Flow, a static type checker for JavaScript, version 0.19.1
> flow
test.es:3
  3: import EventEmitter from 'events'
                              ^^^^^^^^ events. Required module not found
```

Copying the `node.js` definition file from https://github.com/facebook/flow/blob/master/lib/node.js, putting them into `interfaces/` and then referencing it via `[libs]` configuration yields the same error.

I tried this on a small project and couldn’t repro. Under which circumstances could this happen?
